content/content-strategy/define-user-content-needs/create-actionable-artefacts/journey-maps/supporting-takeaway1.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: text/callout
+---
+### Materials
```

Comment:
Duplicate content. The way the other file has been set up is the better way to go. Should remove this file.

Reply:
Steve fixed